### DISCUSSION TITLE
VB Compiler Optimisations

### DISCUSSION THREAD

**Discussion by dmrvb** [twinbasic/twinbasic#196] [Ideas]
A request for when you get on with the optimised Compiler.......
I sometimes use the VB compiler optimisations.
Turning off array bounds checks and numeric overflow checks can significantly increase execution speed on some code.
The big danger is that it leaves other code vulnerable to untrapped errors (had a nasty bug to sort out just last week!)

Could we have these opimisations selectable on a per routine basis, or at least on a per module basis?
So some sort of compiler directive than can be including in the code?

**Comment by WaynePhillipsEA** [comment]
Yes, we'll be able to offer that via attributes, see #25.
